src/main/java/slothingwaffler/Task.java

```diff
@@ -0,0 +1,56 @@
+package slothingwaffler;
+
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a task in the application.
+ * <p>
+ * This is an abstract class representing a task with a description and a status indicating whether it's done.
+ * </p>
+ */
+public abstract class Task {
+
+    // For now, Deadline Tasks will take in dates only while Event Tasks will take in dates and times
+    protected static final DateTimeFormatter DTF1 = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+    protected static final DateTimeFormatter DTF2 = DateTimeFormatter.ofPattern("MMM dd yyyy HH:mm");
+    protected static final DateTimeFormatter DTF3 = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+    protected static final DateTimeFormatter DTF4 = DateTimeFormatter.ofPattern("MMM dd yyyy");
```

Comment:
perhaps might be good to add a line break after this to introduce logical separation